app/src/main/java/org/jellyfin/androidtv/util/BackgroundManagerUtils.kt

```diff
@@ -0,0 +1,37 @@
+package org.jellyfin.androidtv.util
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import androidx.leanback.app.BackgroundManager
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+import timber.log.Timber
+
+fun BackgroundManager.drawable(
```

Comment:
As this file only contains a single extension function I'd recommend renaming it to "BackgroundManagerExtensions" to stay consistent with the other file(s)